docs/scripting/what-is-script.md

```diff
@@ -4,10 +4,23 @@ category: General
 tags:
     - experimental
 mentions:
-	- cda94581
+	- ConsoleTerm
 ---
 
 ::: warning
-The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft beta 1.19.60.26
+The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft beta 1.19.70
 :::
 
+## API
+API is stands for (Application Programming Interface), it is a kind of establishing contacts between two applications, the term application contains a lot more than you might imagine under the term application, it can be a simple script or a game or an entire operating system. The API that you encounter the most is right in front of you, and that API is between the application and your operating system, for example with the cursor on the screen, why is it related to the API? Because your application can find out through the API where the cursor is on the screen without having to implement its own cursor, this means that Minecraft also uses a lot of Wrappers and they then use the API system directly so that we can see the application. Summary:
+An API is a kind of mutual or one-sided interaction between two or more Applications!
+
+## Script API
+
+Maybe you have already heard Script API somewhere, it is important to know what is hidden under this name. A lot of people imagine different things like Codes, Scripts, Programming Languages, (`Mysterious Experiments in Minecraft? :]` ), but in reality it is only, API is not a programming language, script, code, or program, it is a set of functions that allow us to APIs give us the possibility to interact with Minecraft as such, we can receive events, read blocks, entities, create particles, through scripts! This API is also well documented in official web site [here](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server/minecraft-server)!
```

Comment:
This API is also well documented in [official web site](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server/minecraft-server)!